.config/nextest.toml

```diff
@@ -0,0 +1,88 @@
+[store]
+# The directory under the workspace root at which nextest-related files are
+# written. Profile-specific storage is currently written to dir/<profile-name>.
+dir = "target/nextest"
+
+# This section defines the default nextest profile. Custom profiles are layered
+# on top of the default profile.
+[profile.default]
+# "retries" defines the number of times a test should be retried. If set to a
+# non-zero value, tests that succeed on a subsequent attempt will be marked as
+# non-flaky. Can be overridden through the `--retries` option.
+# Examples
+# * retries = 3
+# * retries = { backoff = "fixed", count = 2, delay = "1s" }
+# * retries = { backoff = "exponential", count = 10, delay = "1s", jitter = true, max-delay = "10s" }
+retries = { backoff = "exponential", count = 5, delay = "1s", jitter = true }
```

Comment:
It's built-in to nextest too? 🤯

Reply:
Since it wasn't clear to me from these numbers and I had to look at the docs at https://nexte.st/book/retries.html?highlight=backoff#exponential-backoff: the base is always 2, i.e., `t = delay * 2^count`. So the max delay here is 32 seconds.

I'm not sure if there's any benefit of using exponential backoff here as opposed to simple fixed delay, but I think that's fine too.

Reply:
I started with fixed as well. I don't have a strong opinion either we can vary it based on how many flakey tests exist and what delay to they introduce. Jitter should only account for a few millis.

Reply:
That could explode CI run time quite a bit if we break a lot of tests in PR. 50 broken tests = extra 26 minutes. And when doing big changes it's pretty common for us to start with a bunch of tests broken and work upwards from that. So, I'd rather start with fixed delay now and see how it goes.

Reply:
Ok, done in a4aafe468c72dc9fe35d8dbd1d4faea5ce7bc71d